README.md

```diff
@@ -26,6 +26,7 @@ It’s available in English :gb: and Russian :ru:.
 * This plugin provides Music control buttons (Play/Pause, Next, Previous, Like/Unlike) :musical_note:
 * and Player information (track and artist name, album art)
 * It includes Keyboard Shortcuts
+* Integration with operating system multimedia controls. Which also allows you to use a multimedia keyboard
```

Comment:
> Do you mean multimedia keys on a keyboard?

Yep, exactly. The idea is to control YaMusic while the browser is not focused or even minimized. Tt is the easiest way to do that I found.
Also it integrates with operation system and allows you control YaMusic even without keyboard.

https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API

For example, in windows (pic from Google):
![image](https://user-images.githubusercontent.com/2737339/104028367-af8df000-51d9-11eb-8e66-6ddb633c5c34.png)

In XFCE (my screenshot):
![image](https://user-images.githubusercontent.com/2737339/104028621-fe3b8a00-51d9-11eb-8625-c1e375c32f94.png)

Reply:
Looks great, thanks a lot! I think we need to tell about this feature to users on Settings page instead of `Unfortunately, Firefox doesn’t support multimedia keys yet`.

Also we can add a hint:

<img src="https://amplifr-direct.s3.eu-west-1.amazonaws.com/social_images/image/c9b3e441-2503-4c8e-a4a3-06a13a54c4a8.png" alt="hint screenshot" />

I think it doesn’t block merging, and we can add it later in the next release if you wish. Users ask for global hotkeys sometimes, and it might help.

Reply:
> Also we can add a hint:
Do you mean that if user's platform doesn't support Media Session API, he should get notification about that?

Reply:
Nope, I mean, we can show a notification that the plugin supports Media Session API now. But the user most likely doesn’t know what is Media Session API.

I’ve tested the plugin on Mac today and I think now that the hint is useless because global hotkeys work out-of-the-box. I supposed that it requires to install some software but it’s not necessary.